### PR TITLE
updating compatibility for onpopstate event

### PR DESCRIPTION
**Pull request by a1626**
source [CanIUse](https://www.caniuse.com/#search=popstate)

### CODE REVIEW

**Comment by a1626** [comment]
@Elchi3 sorry my bad. I did these changes few weeks back and forgot to send PR at that time, and i thought that i took data only after using the show all button, but it seems like that's not the case.

File: api/WindowEventHandlers.json [hunk 1]
```diff
@@ -496,37 +496,37 @@
...
             "edge_mobile": {
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": "61"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "60"
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply api/WindowEventHandlers.json]: This is still 60. It should be 4 to match the desktop version.

**Comment by a1626** [comment]
@Elchi3 i've updated versions for chrome and firefox android but i still have a doubt. Can you share with me how did you discovered that they were the first versions that supported `popstate` event?

**Comment by Elchi3** [comment]
@a1626 Thank you! These are the first versions of the browsers that have been released. See https://github.com/mdn/browser-compat-data/tree/master/browsers for release data.

**Pull request merged**